manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+PackageLocale: en-US
+Publisher: Beyond All Reason
+PackageName: Beyond-All-Reason
+License: copyrighted

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
I could help you figure out this.

Some parts are as open as MIT or cc-by. Others are proprietary/copyrighted.

So overall I think that this whole package should be `Proprietary`.

Reply:
I looked through the project repo and website, and couldn't find licensing details beyond "free to download and install". Unfortunately, no `License.txt` was present.